unimath-symbols.ltx

```diff
@@ -4,10 +4,10 @@
 % SYMBOLS DEFINED BY UNICODE-MATH
 %%%%%%%%%%%%%%%%%%%%%%%
 
-\documentclass{article}
+\documentclass[final]{article}
 \makeatletter
 
-\usepackage{booktabs,catchfile,shortvrb,geometry,metalogo,varwidth,textcomp,longtable,hyperref}
+\usepackage{ragged2e,setspace,booktabs,catchfile,shortvrb,geometry,metalogo,textcomp,longtable,tabu,hyperref}
```

Comment:
It's pretty necessary unless you can point me towards another package that provides `X` columns (or the equivalent) compatibly with `longtable`.